### PR TITLE
Refs/heads/bc 8179

### PR DESCRIPTION
**Pull request by atlldwp**
# Description

## Links to Tickets or other pull requests
BC-8179 related.

## Changes
- Added data-testids on some elements related to the user registration process.

## Approval for review

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### CODE REVIEW

File: views/lib/forms/paginatedForm.hbs [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 	<div class="stages">
 		{{#for 1 (add sections 1) 1}}
-			<label role="button" class="section-button" aria-pressed="false" tabindex="0" for="section-{{this}}">{{this}}</label>
+			<label role="button" class="section-button" aria-pressed="false" tabindex="0" for="section-{{this}}" data-testid="sectionButton-{{this}}">{{this}} </label>
```
Comment (MartinSchuhmacher) [inline views/lib/forms/paginatedForm.hbs:10]: please be more consistent with styling here, e.g. `section-button-{{this}}`

File: views/registration/registration-student.hbs [hunk 2]
```diff
@@ -93,8 +93,8 @@
                 <div class="print">
                     <div class="recap">
                         <!--<p>Nutzername: <i data-from="firstName"></i>.<i data-from="lastName"></i></p>-->
-                        <p>{{$t "global.label.emailAddress"}} <i data-from="email"></i></p>
-                        <p>{{$t "global.label.password"}} <i class="student-password">{{password}}</i></p>
+                        <p>{{$t "global.label.emailAddress"}} <i data-from="email" data-testid="student-email"></i></p>
+                        <p>{{$t "global.label.password"}} <i class="student-password" data-testid="student-initial-pwd">{{password}}</i></p>
```
Comment (MartinSchuhmacher) [inline views/registration/registration-student.hbs:97]: `student-initial-password`

**Review by MartinSchuhmacher — CHANGES_REQUESTED** [review]

File: views/lib/forms/paginatedForm.hbs (hunk 1 above)
Comment (atlldwp) [reply views/lib/forms/paginatedForm.hbs]: Updated it.

File: views/registration/registration-student.hbs (hunk 2 above)
Comment (atlldwp) [reply views/registration/registration-student.hbs]: done!

**Review by MartinSchuhmacher — APPROVED** [review]

**Pull request merged**